src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,479 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Content Collections (Experimental)
+description: Content collections help organize your Markdown and type-check your frontmatter with schemas.
+i18nReady: false
+---
+
+Content collections help organize your Markdown or MDX and type-check your frontmatter with schemas. You may reach for collections if you:
+- **Plan to use content in multiple areas** of your site (landing pages, footers, navigation, etc).
+- **Have a non-trivial number of documents** to manage and fetch (example: a blog with 50+ posts).
+- **Want to enforce frontmatter fields,** and fail if fields are missing (example: every blog post should have a title and description).
```

Comment:
I'm good with the secon edit, but I'll push back on "large number of documents" per Erika's comment. 50+ posts is considered small to some developers, and medium to others. I think "non-trivial" covers the general idea that once you notice how many documents you have, collections are helpful!

Reply:
I will once again push back on "non-trival." It's not great language for docs. Can we come up with something that is more straight-forward English, less vague, and translates to multiple languages more easily?

Some options could include:

> Start to have trouble managing your documents

> Discover that querying and managing all documents is difficult

Maybe something along those lines? Any thoughts?

Reply:
Ah, or better “Have a growing number of documents.” Implies you have more to manage without judging the size that’s right for switching

Reply:
Ben, can you say which _features_ of Content Collections make more sense as document count grows/is “large”? Is it just that with larger collections, the benefits of frontmatter consistency grows? Or is their some performance gain?

We seem to be having trouble pinning this bullet point down and I wonder if it’s because we’re not actually clear of the benefits for this specific use case.